inventory/group_vars/all.yml

```diff
@@ -50,10 +50,11 @@ swapfile_size: false
 
 #----------------------------------------------------------------------
 # Rails variables
-ruby_version: 2.3.7
+ruby_version: 2.4.4
```

Comment:
This doesn't actually have an impact on the server, does it? Shall we just select the last element in `ruby_versions` for this?

Reply:
I think this might set the local rbenv version to 2.4.4 which is probably not what we want...

Reply:
That was my first thought too, that this would **set** the ruby version to 2.4.4; but the impression I got from Luis's comment and https://github.com/openfoodfoundation/ofn-install/pull/549 is that it only installs it. But, uh, yeah, we should probably confirm this before merging 😂

Reply:
Ah yes, maybe it's ok. Let give it a quick try before we merge though.

Reply:
I have a feeling that when we merged #549, the two versions were both basically "working" without causing fatal errors (which is not the case here) and we were merging the update to master in the main repo around the same time (which is also not the case here).